Dockerfile

```
@@ -0,0 +1,9 @@
+FROM python:3.12.0-alpine3.18
```

why did you specify the version of the image with so much detail in a template?

```suggestion
FROM python:alpine
```